codebuild/spec/buildspec_integv2_nix.yaml

```diff
@@ -0,0 +1,272 @@
+        variables:
+          INTEGV2_TEST:
+            happy_path client_authentication sni_match buffered_send npn
+            signature_algorithms ocsp external_psk pq_handshake serialization
+          NIXDEV_ARGS: --max-jobs auto .#awslcfips2024
+          NIX_CACHE_BUCKET: s3://codebuildnixinteg-prod-nixcachebucketintegprodaarc-rqyksjxh6wxa?region=us-east-2
+    - identifier: Integ_awslcfips2024_aarch64_1
+      depend-on:
+        - nixCache_aarch64
+      env:
+        compute-type: BUILD_GENERAL1_XLARGE
+        image: public.ecr.aws/l1b2r3y5/nix-aws-codebuild-aarch64:next
+        privileged-mode: true
+        type: ARM_CONTAINER
+        variables:
+          INTEGV2_TEST:
+            renegotiate cross_compatibility early_data hello_retry_requests
+            fragmentation key_update session_resumption version_negotiation
+          NIXDEV_ARGS: --max-jobs auto .#awslcfips2024
+          NIX_CACHE_BUCKET: s3://codebuildnixinteg-prod-nixcachebucketintegprodaarc-rqyksjxh6wxa?region=us-east-2
+
+    # Openssl30 x86
+    - identifier: Integ_openssl30_x86_0
+      depend-on:
+        - nixCache_x86_64
+      env:
+        compute-type: BUILD_GENERAL1_XLARGE
+        image: public.ecr.aws/l1b2r3y5/nix-aws-codebuild:latest
+        privileged-mode: true
+        type: LINUX_CONTAINER
+        variables:
+          INTEGV2_TEST:
+            happy_path client_authentication sni_match buffered_send npn
+            signature_algorithms ocsp external_psk pq_handshake serialization
+          NIXDEV_ARGS: --max-jobs auto .#default
+          NIX_CACHE_BUCKET: s3://codebuildnixinteg-prod-nixcachebucketintegprodx861-ehnvuoswh2yr?region=us-east-2
+    - identifier: Integ_openssl30_x86_1
+      depend-on:
+        - nixCache_x86_64
+      env:
+        compute-type: BUILD_GENERAL1_XLARGE
+        image: public.ecr.aws/l1b2r3y5/nix-aws-codebuild:latest
+        privileged-mode: true
+        type: LINUX_CONTAINER
+        variables:
+          INTEGV2_TEST:
+            renegotiate cross_compatibility early_data hello_retry_requests
+            fragmentation key_update session_resumption version_negotiation
+          NIXDEV_ARGS: --max-jobs auto .#default
+          NIX_CACHE_BUCKET: s3://codebuildnixinteg-prod-nixcachebucketintegprodx861-ehnvuoswh2yr?region=us-east-2
+
+    # Openssl30 aarch
+    - identifier: Integ_openssl30_aarch64_0
+      depend-on:
+        - nixCache_aarch64
+      env:
+        compute-type: BUILD_GENERAL1_XLARGE
+        image: public.ecr.aws/l1b2r3y5/nix-aws-codebuild-aarch64:next
+        privileged-mode: true
+        type: ARM_CONTAINER
+        variables:
+          INTEGV2_TEST:
+            happy_path client_authentication sni_match buffered_send npn
+            signature_algorithms ocsp external_psk pq_handshake serialization
+          NIXDEV_ARGS: --max-jobs auto .#default
+          NIX_CACHE_BUCKET: s3://codebuildnixinteg-prod-nixcachebucketintegprodaarc-rqyksjxh6wxa?region=us-east-2
```

Comment:
All your NIXDEV_ARGS have "--max-jobs auto" and only differ by the libcrypto. Does the "--max-jobs auto" need to be repeated every time?

Reply:
I've split these into two variables to avoid the duplication.